AGPs/The-Aragon-Network-is-not-Game-of-Thrones.md

```diff
@@ -0,0 +1,38 @@
+---
+## AGP: N/A
+Title: AN ≠ GoT
+Author: burrrata (@burrrata)
+Status: Stage III
+Track: Meta
+Created: 2019-11-19
+---
+
+# AGP-X: AN ≠ GoT
+
+## File changed (AGP-0 or AGP-1)
+
+AGP-1
+
+## Link to proposed change
+
+https://github.com/aragon/AGPs/pull/146
+
+## Motivation for making this change
+
+AGP-103 was marketed as a way to minimize spending by enforcing a budget. This is accomplished via a 2m cap on finance-track spending, however, this is not the only consequence of this proposal. 
+
+AGP-103 favors recurring subscriptions with the maximum amount of absolute support. It also favors current recurring proposals over new proposals. Let's explore what that really means...
+
+This incentivizes teams to ask for as much money as possible as early as possible. It's a SaaS model, but for Aragon treasury funding. It's easy to sign up for (ANT holders vote yes on almost everything), and hard to cancel (to date an AGP has never been reversed).
+
+AGP-103 favors current recurring payments over new proposals asking for funds. The teams who are currently in the Aragon ecosystem will benefit greatly from this. New teams will be discouraged to get involved in the network. If they do want to get involved they will have to find a way to make room in the budget for their projects. 
+
+To address the fact that AGPs have never been reversed new proposals cancelling currently recurring proposals will be favored before all others. Note: this does not mean that a team has to cancel their own proposal. This means that any proposal that cancels another proposal will be processed before all others. Anyone can submit an AGP. This means that every quarter every team (current or new) will be incentivized to submit an AGP cancelling the funding of other teams. They can do this transparently or anonymously. 
+
+On top of all that, AGP-103 favors proposals with the most absolute support. Currently the majority of active ANT voters are very centralized. A few people and teams have more weight than anyone else. This is far from decentralized. This is not the utopian future Aragon promised. The Aragon Network needs to be a fair and open place for people to collaborate in positive sum games. AGP-103 creates a zero sum game. This is not [what Aragon is supposed to be about.](https://github.com/aragon/AGPs/blob/master/AGPs/AGP-0.md)
```

Comment:
```suggestion
This is not the utopian future Aragon promised.
```

Has "Aragon" ever promised utopia (a word whose roots literally mean "no place")? Have governance decisions ever been anything other than tokenholder-driven?

> AGP-103 creates a zero sum game.

I don't think it does. Nothing in AGP-103 says that authors who wish to spend outside of the budget cannot propose an AGP that increases the budget so they can spend more. That said, there is a limited pool of funds to draw from. The AA/ AN does not have infinite money. And so in that larger sense, there is a zero-sum game, but AGP-103 did not create that, that is just reality.